### ISSUE TITLE
Blockchain projects should be possible to add as a category

### ISSUE THREAD

**Issue by sharppanda** [graphprotocol/mission-control-curator#242]
The Everest lists dApps from multiple blockchains. It would be much clearer if a blockchain project could also serve as a category. That way, any dApps can select which blockchain it supports.

**Comment by yanivtal** [comment]
Yes I think **Network** should be a field of **dApps** and **Protocols**. I wonder if **Network** should be a field of **Project**. A Project for example could have multiple applications on different networks.

**Comment by sharppanda** [comment]
Everything in Everest is a project now. Where is the difference between dApps / Protocols / Projects?

The ideal would be if a project could have 0-n networks selected (list taken from [Science & Tech -> Software -> Blockchain](https://everest.link/category/0x18c98c943fd9aab3e0a277b23350b8b975df85ff/).)
